### PR TITLE
Max 16

### PR DESCRIPTION
**Pull request by MassimoVilla**
Here comes the last-but-one MS of the Juel-Jensen collection: MS BDLaethg38. It is a scroll with protective texts

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
Thank you! I carried out a few minimal corrections directly in the file.

File: OxfordBodleian/Juel-Jensen/BDLaethg38.xml [hunk 1]
```diff
@@ -50,60 +108,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                <item xml:id="e4">
+                                    <desc>Brillenbuchstaben <!--italics--> are incorporated within the text into several places of <ref target="#ms_i2 #ms_i3 #ms_i5">textual units II, III, and V</ref>.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
                         
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">The three parchment strips that compose the scroll are sewn together by means of narrow parchment strips.</decoNote>
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethg38.xml:225]: Perhaps, can we add (...) sewn together "with a running stitch"? (...)

**Review by ElianaDalSasso — CHANGES_REQUESTED** [review]
Thank you Massimo! For me it's good! I have suggested just a minor addition in the binding

File: OxfordBodleian/Juel-Jensen/BDLaethg38.xml (hunk 1 above)
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg38.xml]: Thank you Eliana, modified.
Feel free to expand the binding part if you feel appropriate

**Pull request merged**